build.rs

```diff
@@ -0,0 +1,9 @@
+fn main() {
+    println!("If the build fails pass the llvm lib folder to env LLVMLIB_DIR");
+    let lib_dir = std::env::var_os("LLVMLIB_DIR");
+    if lib_dir.is_some() {
+        let reallib1 = lib_dir.unwrap_or_default();
+        let reallib2 = reallib1.to_str().unwrap_or_default();
+        println!("{}",format!("cargo:rustc-link-search={llvmLibs}", llvmLibs = reallib2));
```

Comment:
```rust
        println!("If the build fails pass the llvm lib folder to env LLVMLIB_DIR");
        if let Some(name) = std::env::var("LLVMLIB_DIR") {
            println!("cargo:rustc-link-search={}", name);
        }
```

is much more concise :)

Reply:
thanks I wont lie I did this with little rust experience. Coming from experience in other languages I kind of tried to find a middle ground between rust specific features and other languages.

Reply:
In the future you can use `cargo clippy`, it will suggest a lot of helpful tips to improve your code and to avoid anti-patterns :)